scripts/find_outliers.py

```diff
@@ -22,6 +26,196 @@ def find_outliers(data_directory):
     None
     """
     # Your code here
+
+    ## MLN:  10-5-16
+
+    #make an empty string array that will ultimately store which volumes
+    #are outliers
+    outlierarr = []
+
+    # Identify image files in data_directory using names listed in
+    # 'hash_list.txt' (executed similarly to validate_data.py)
+    hashlist = str(data_directory + '/' + 'hash_list.txt')
+
+    #iterating through to add to an array which contains the values for each volume for each files
+    k = 0
+    vol_means = {}
+    volumes = {}
+    imgfiles = {}
+
+    for line in open(hashlist, 'rt'):
+        i = line.split()
+        imgfile = str(data_directory + '/' + i[1]) #making this filename a string with full path
```

Comment:
Actually, you don't need the `str` here - it's already a string.   Could use `os.path.join(data_directory, i[1])` so it also works on Windows.